api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -20,12 +25,15 @@
 import org.pmiops.workbench.exceptions.BadRequestException;
 import org.pmiops.workbench.exceptions.ForbiddenException;
 import org.pmiops.workbench.model.AccessModule;
+import org.pmiops.workbench.model.AccessModuleStatus;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class AccessModuleServiceImpl implements AccessModuleService {
   private static final Logger logger = Logger.getLogger(AccessModuleServiceImpl.class.getName());
+  private static final long MIN_ACCESS_EXPIRATION_EPOCH_MS =
```

Comment:
since we have now passed this time, it's probably better to delete it.

Reply:
I keep it because I am not sure if that will break things. Code I am not sure is: https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java#L270
If removing  MIN_ACCESS_EXPIRATION_EPOCH_MS, API may return different value(e.g. anytime before that would becomes the really expiration time instead of the MIN_ACCESS_EXPIRATION_EPOCH_MS). 
If our UI does not use that value to render information, or we are fine to use the older timestamp, then we can remove that.

Reply:
I think it's probably OK to remove... but I'll do it in a standalone PR so we can analyze it in isolation